docs/modules/policies/pages/authoring_tips.adoc

```diff
@@ -79,3 +79,44 @@ use({
     end,
 })
 ----
+
+==== JetBrains IDEs
+
+Go into the Preferences of JetBrains IDE of choice and navigate to the `JSON Schema Mappings` menu.
+
+----
+Preferences > Languages & Frameworks > Schemas and DTDs > JSON Schema Mappings
```

Comment:
I think you can use the Menu macro here: https://docs.asciidoctor.org/asciidoc/latest/macros/ui-macros/#menu-macro-syntax